internal/server/http/handler/v1/watering_plan/handler.go

```diff
@@ -33,12 +34,12 @@ var (
 // @Router			/v1/watering-plan [get]
 // @Param			page	query	string	false	"Page"
 // @Param			limit	query	string	false	"Limit"
```

Comment:
Page and limit are integers, right? We can specify types in Swagger
```suggestion
// @Param			page	query	int	false	"Page"
// @Param			limit	query	int	false	"Limit"
```